homepage/static/css/landing.css

```diff
@@ -0,0 +1,97 @@
+img.logo {
+	width: 260px;
+	height: 260px;
+	margin: 70px !important;
+}
+
+.page-container {
+	justify-content: center;
+	display: flex;
+	flex-direction: column;
+	padding: 0 !important;
+}
+
+.top-container {
+	background-color: #182B43;
+	color: #FE812D;
+	display: flex;
+	flex-direction: column;
+	flex-grow: 1;
+	align-items: center;
+	box-shadow: 2px 2px 3px #FE812D;
+}
+
+.top-content {
+	display: flex;
+	flex-direction: row;
+}
+
+.top-title{
+	display: flex;
+	flex-direction: column;
+	align-self: center;
+	text-shadow: 3px 3px 3px #888888;

```

Comment:
Suggestion: a lighter tone for the text-shadow? Or make the shadow wider, maybe; because when looking at it and trying to read it gets a bit headache-y

( found this fantastic website for hex colors: [colorhexa](https://www.colorhexa.com/888888))

Reply:
will do. :) 
though it looks worse than it is in the small display

Reply:
@VeraTs I changed it, wdyt?

Reply:
Looks Great!